src/main/resources/hudson/plugins/git/GitBranchTokenMacro/help.jelly

```diff
@@ -14,8 +14,8 @@
     </dd>
     <dt>fullName</dt>
     <dd>
-      If specified, this token expands to the full branch name, such as 'origin/master'.
-      Otherwise, it only expands to  the short name, such as 'master'.
+      If specified, this token expands to the full branch name, such as 'refs/remotes/origin/master'.
+      Otherwise, it only expands to the short name, such as 'remotes/origin/master'.
```

Comment:
@KostyaSha Here I changed the docs of the already existing GIT_BRANCH (not GIT_BRANCH_SHORT) because docs were not updated after implementation change. see JENKINS-25465 for details.